### PR TITLE
Added sep (character) option to import_matrix_table.

### PR DESCRIPTION
**Pull request by cseed**

### CODE REVIEW

File: src/main/scala/is/hail/HailContext.scala [hunk 1]
```diff
@@ -584,12 +585,14 @@ class HailContext private(val sc: SparkContext,
...
-    forceBGZ: Boolean): MatrixTable = {
+    forceBGZ: Boolean,
+    sep: String = "\t"): MatrixTable = {
+    assert(sep.length == 1)
 
     val inputs = hadoopConf.globAll(files)
 
     forceBGZip(forceBGZ) {
-      LoadMatrix(this, inputs, rowFields, keyNames, cellType = TStruct("x" -> cellType), missingVal, nPartitions, noHeader)
+      LoadMatrix(this, inputs, rowFields, keyNames, cellType = TStruct("x" -> cellType), missingVal, nPartitions, noHeader, sep(0))
```
Comment (catoverdrive) [inline src/main/scala/is/hail/HailContext.scala]: can you enforce here that `sep` is only one character long?

File: python/hail/methods/impex.py [hunk 2]
```diff
@@ -1309,8 +1311,11 @@ def import_matrix_table(paths,
         raise FatalError("""import_matrix_table expects entry types to be one of: 
         'int32', 'int64', 'float32', 'float64', 'str': found '{}'""".format(entry_type))
 
+    if len(sep) != 1:
```
Comment (cseed) [inline python/hail/methods/impex.py]: @catoverdrive I enforce that sep is 1 character here.

File: src/main/scala/is/hail/HailContext.scala (hunk 1 above)
Comment (cseed) [reply src/main/scala/is/hail/HailContext.scala]: I enforce in python here: https://github.com/hail-is/hail/pull/3444/files#r184263112

Comment (catoverdrive) [reply src/main/scala/is/hail/HailContext.scala]: oops. thanks!

**Pull request merged**